packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -40,7 +40,7 @@ export class CallContext {
   private _emitter: EventEmitter;
   private _atomicId: number;
 
-  constructor(userId: string, maxListeners?: number) {
+  constructor(userId: string, maxListeners = 50) {
```

Comment:
Thanks Nan for fixing this. I missed this when convert the hardcoded value to an option. For this could we instead pass in the 50 to the createStatefulCallClient as a parameter? For example:
```
const newStatefulCallClient = createStatefulCallClient({ userId: userIdFromToken }, { maxStateChangeListeners: 50 });
setStatefulCallClient(newStatefulCallClient);
```

Reply:
Thanks Allen, the opinion here is client should provide some default options to make it functional well in default scenarios, so at least it should support each component to register its own usePropsFor(so 50 by default is a reasonable number). By default, people don't have to care about it when they create client (90% of time)

But we do need to have a place to document this maxListeners parameter and let devs know about the existence of this, if people have a bug that re-render too much or they just register onChange event for too many times, they will easily find a document to debug.